### PR TITLE
Remove custom webhook and add snapshot controller and webhook

### PR DESCRIPTION
**Pull request by kfox1111**
This change implements https://github.com/vsphere-tmm/helm-charts/issues/47. It enables the snapshot webhook to be enabled/disabled as needed along with managing the snapshot controller from the same chart.

### CODE REVIEW

File: charts/vsphere-csi/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,15 @@
 
 apiVersion: v2
-appVersion: 2.5.1
+appVersion: 2.6.0
```
Comment (sathieu) [inline charts/vsphere-csi/Chart.yaml:3]: You probably want to update `version`  instead.

**Comment by sathieu** [comment]
@kfox1111 Could you please rebase and resolve conflicts 🙏 ?

**Comment by kfox1111** [comment]
Rebased.

**Comment by sathieu** [comment]
@MaxRink @mylesagray Please review this PR by @kfox1111 :christmas_tree: .

**Pull request merged**